packages/app-mobile/components/screens/ShareManager/AcceptedShareItem.tsx

```diff
@@ -64,7 +64,7 @@ const AcceptedShareItem: React.FC<Props> = props => {
 		} catch (error) {
 			logger.error('Failed to leave share', error);
 			await shim.showMessageBox(
-				_('Failed to leave share. Please verify that Joplin is connected to the internet and able to sync.\nError: %s', error),
+				_('Failed to leave shared notebook. Please verify that Joplin is connected to the internet and able to sync.\nError: %s', error),
```

Comment:
Actually I think we should try to keep our error messages a bit simpler to make translation easier

Most of them will never be seen by the user and the error that was thrown should be sufficiently descriptive and if it's not that's when we start improving error messages.

But we should start with something simple, like I've done in the desktop implementation. Just `Error: %s` is enough most of the time